.github/ISSUE_TEMPLATE/depandabot.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright The ORAS Authors.
```

Comment:
I'm prettu sure this file should be in the `.github` directory

Reply:
Your first commit is not signed off and I think that is causing the DCO problem. You might want to squash the 3 commits into one like https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together and `git commit -signoff` on the one commit. After that force push the changes